### ISSUE TITLE
Bug when showing step log

### ISSUE THREAD

**Issue by dactt** [wenwenyu/PICK-pytorch#91]
I'm pretty sure the result of loss showing in training step log is loss of only one batch before log showing step
